ILRepack/ILRepack.cs

```diff
@@ -352,7 +352,9 @@ public void Repack()
                 Logger.Info("Output directory does not exist. Creating output directory: " + outputDir);
                 Directory.CreateDirectory(outputDir);
             }
-            TargetAssemblyDefinition.Write(Options.OutputFile, parameters);
+            using (FileStream outputStream = File.OpenWrite(Options.OutputFile))
+                TargetAssemblyDefinition.Write(outputStream, parameters);
```

Comment:
@timotei Cecil should have resolved that, http://cecil.pe/post/151940968861/disposable-ownership but 0.10 is still in beta.